WordPress/src/main/res/values/backup_styles.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Good idea to create the styles in a separate file! It might come handy if we decide to move the backup into a different module.